Project.toml

```diff
@@ -33,7 +35,7 @@ JSON = "0.21"
 LightGraphs = "1"
 PyCall = "1"
 SimpleWeightedGraphs = "1"
-julia = "1.4, 1.5, 1.6"
+julia = "1.6"
```

Comment:
Are either Xtals or NearestNeighbors not compatible with 1.4 and 1.5?

Reply:
Xtals needs 1.6, yeah